app/assets/stylesheets/fortitude/components/_tabs.scss

```diff
@@ -4,7 +4,18 @@ title: Tabs
 name: tabs
 category: Objects
 ---
-The `tabs` object is used to encapsulate content in individual tabs.
+The `tabs__content` object is used to encapsulate the content for individual tabs. See [the javascript component](/javascript.html#js-tabs) for info on how to combine `tabs` , `tabs__content` and `tabs-navigation` to show and hide tabs.
```

Comment:
probably needs a refactor after talking about the shown/hidden states we talked about on slack.